dnf.spec

```diff
@@ -43,6 +43,7 @@ Requires:       python3-%{name} = %{version}-%{release}
 %else
 Requires:       python2-%{name} = %{version}-%{release}
 %endif
+Recommends:     (python%{?with_python3:3}-dbus if NetworkManager)
```

Comment:
Yum has had support for Recommends and Suggests since `yum-3.4.3-508`, which shipped in Fedora 22.

Langpacks use Supplements, not Recommends.